pkg/util/encryption/xchacha20poly1305.go

```diff
@@ -53,3 +54,7 @@ func (c *xChaCha20Poly1305) Seal(input []byte) ([]byte, error) {
 
 	return append(nonce, c.aead.Seal(nil, nonce, input, nil)...), nil
 }
+
+func (c *xChaCha20Poly1305) GetSealSecretVersion() string {
+	return c.secretVersion
```

Comment:
where is it comming from? We do not set `secretVersion` anywhere as far as I see.

Is it supposed to be an empty string? If so, should we moreve the field and `return ""` here?